spec.yaml

```diff
@@ -88,6 +91,115 @@ paths:
       responses:
         '200':
           description: Successfully uploaded content
+
+  /repos/{owner}/{repo}/storage:
+    get:
+      tags:
+        - Storage Integrations
+      summary: List integrated storages in the repository
+      operationId: getBuckets
+      parameters:
+      - $ref: './components/parameters/Owner.yaml'
+      - $ref: './components/parameters/Repository.yaml'
+      responses:
+        '200':
+          description: Successfully listed integrated storages
+          content:
+            application/json:
+              schema:
+                $ref: './components/schemas/StorageIntegration.yaml'
+              example:
+                [
+                  {
+                    "name": "bucket/prefix",
+                    "protocol": "s3",
+                    "list_path": "https://dagshub.com/Simon/baby-yoda-segmentation/dataset/storage/content/s3/bucket/prefix"
```

Comment:
Since it's a part of the example, it should be an actual link without placeholders.

Reply:
We don't really have a showcase repo for storage buckets :/
SavtaDepth has one hooked up, but it's actually the dvc cache, so it's kind of ugly looking

Reply:
Dean gave me a great idea to use a dataset repo, so I'll do that